src/server/db/migrations/20160217083000-simsupplier-migration.test.js

```diff
@@ -38,18 +38,12 @@ module.exports.up = function(db, config)
 
     // -----
 
-    // Load data.
-    const user2SupplierData = require(path + '/user2supplier.json');
```

Comment:
Even if I add it in another migration file I will still get an error. What is the reason for having another migration file for test data btw? test images could always be deleted and rebuilt